test/config/led-save-group.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why this change?

Reply:
I got this as part of rebase, after you revoked 1 commit. but I don't think we need it